bench/ibench/autovac_driver.py

```diff
@@ -119,7 +121,25 @@ def startExp(self, env_info):
         self.db_host = env_info['db_host']
         self.db_user = env_info['db_user']
         self.db_pwd = env_info['db_pwd']
-        self.table_name = env_info['table_name']
+
+        collectExperimentParams(self.env_info)
+        self.initial_size = self.env_info['initial_size']
```

Comment:
Can we separate refactoring from functional changes? Those can go after the refactor and probably only to your branch.